hr_expense_cancel/hooks.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+
+        for sheet in env['hr.expense.sheet'].search([]):
+            if sheet.expense_line_ids[0].payment_mode == 'own_account':
```

Comment:
There might be sheets without lines. Use `sheet.expense_line_ids[:1]` as a safeguard

Reply:
done

Reply:
Well, I actually did it as you proposed in this  comment https://github.com/OCA/hr/pull/557#discussion_r261306563